src/global-messages/types.ts

```diff
@@ -18,6 +18,7 @@ export enum GlobalMessageContextEnum {
   appServiceDisruptions = 'app-service-disruptions',
   appLogin = 'app-login',
   appLoginPhone = 'app-login-phone',
+  appAnonymousFlexibleTickets = 'app-anonymous-flexible-tickets',
```

Comment:
I think this should be `appTicketingAnonymous` to be consistent with how we name the global message context. From my understanding, the global message can be used on various cases, not only flexible ticket.